### ISSUE TITLE
Ignore unusable identity server

### ISSUE THREAD

**Issue by n1trux** [vector-im/riot-web#9269]
In some (company) networks, vector.im as identity server is blocked by the gateway. Riot doesn't allow logging in without one in that case, as there's a response seemingly by the identity server which is in fact the block notice.

I'd like Riot to show an error that the identity server is not available (maybe with a "what does that mean?" link to an explanation) but still allow logging into my homeserver; albeit without the 3PID feature.

An alternative would be hosting your own identity server, but that's not always wanted or possible. Maybe one could define multiple 3PID servers and Riot picks the first usable one?

Possibly related: #3629, #5921

**Comment by turt2live** [comment]
The spec requires us to validate it (see https://matrix.org/docs/spec/client_server/r0.4.0.html#well-known-uri), but there's nothing stopping you from pointing the client at a webserver which 200 OK's `/_matrix/identity/api/v1` as an empty JSON object.

**Comment by turt2live** [comment]
I'm going to close this as a duplicate of https://github.com/vector-im/riot-web/issues/5921, leaving the bug as https://github.com/vector-im/riot-web/issues/3629
